src/Util/PHP/WindowsPhpProcess.php

```diff
@@ -1,43 +0,0 @@
-<?php
-/*
- * This file is part of PHPUnit.
- *
- * (c) Sebastian Bergmann <sebastian@phpunit.de>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-namespace PHPUnit\Util\PHP;
-
-use PHPUnit\Framework\Exception;
-
-/**
- * Windows utility for PHP sub-processes.
- *
- * Reading from STDOUT or STDERR hangs forever on Windows if the output is
- * too large.
- *
- * @see https://bugs.php.net/bug.php?id=51800
- */
-class WindowsPhpProcess extends DefaultPhpProcess
```

Comment:
now my code depending on this public class will stop working :(

Reply:
Just because a class is `public` does not mean that is part of the public API of a package.

Reply:
damn, without any indicator that it's not part of public API, it's hard to know that :(
so one could rely on it

Reply:
True. But should I really have to add `@internal` to almost all classes?

Reply:
There are 2 ways, either add `@internal` to non-public, or `@api` to public.
Without this info, user cannot know that it's internal, so he assume it's open to use and follows SEMVER.

note: Usually, I mark most classes with `@internal` and I'm making most classes final (so non-internal got final), leaving only few of them open to extend via inheritance.